docs/resources/aws_rds_cluster.md

```diff
@@ -0,0 +1,81 @@
+---
+title: About the aws_rds_cluster Resource
+---
+
+# aws\_rds\_cluster
+
+Use the `aws_rds_cluster` InSpec audit resource to test detailed properties of an individual RDS cluster.
+
+RDS gives you access to the capabilities of a MySQL, MariaDB, PostgreSQL, Microsoft SQL Server, Oracle, or Amazon Aurora database server.
+
+## Syntax
+
+An `aws_rds_cluster` resource block uses resource parameters to search for an RDS cluster, and then tests that RDS cluster.  If no RDS clusters match, no error is raised, but the `exists` matcher will return `false` and all properties will be `nil`.  If more than one RDS cluster matches (due to vague search parameters), an error is raised.
+
+    describe aws_rds_cluster('test-cluster-id') do
```

Comment:
does this work without passing in `db_cluster_identifier:`?

Reply:
Yes this control just requires the value, not the key.